### ISSUE TITLE
Status Update 11/13

### ISSUE THREAD

**Issue by Scrubsauce** [mindaptiv/Foster#121]

**Comment by Simulacrum0** [comment]
1) Can you wrap/catch the exception?  Then bypass it, close down your Win10__Input system ( XINPUT_Close()? ) and then restart it?

I strongly dislike the modern implementation of exception handling due to code-bloat, time/cost in unwilling the stack, and unreliability across modules.  ( overtime, i've come to appreciate setjmp/longjmp and signal raising as efficiently, easy to use ( once a wrapper/package has been made ), and consistent.  Wish the model was supported beyond low-level OS calls and *nix APIs.  Then again, exception handling is easy to add, particularly in languages with first class support for them, so its probably a far superior choice for large projects with many workers, much turnover, etc.

1b) Sounds like trying the new method may yield results as expected?  Well, at least you've got to try it to find out...probably easier/simpler than turning on exception handling ( or does the WinRT API use some Microsoft Exception approach?  SEH vs MS-EH? Can't recall )

Good hunting~

**Comment by Scrubsauce** [comment]
I've tried wrapping the singular XINPUT call I make as well as the overall method and even the update method used for the cube sample app and the exception is never caught.  From looking at the Thread/Stack Frame info in Visual Studio, I think it has to do with the XINPUT core event calls taking place in a separate thread from the main application, but I couldn't say with 100% certainty.  

They're requiring me to use the managed Exception^ object for my try-catch statements as well.  Windows is still finicky over allowing/forbidding different types of classes in different spaces (like we saw with the reference classes two weeks ago).  Going to attempt to write up some new methods for making contact with the new Gamepads namespace and see if I get anything before removing the XINPUT stuff outright.
